pkg/oc/bootstrap/docker/up.go

```diff
@@ -858,7 +852,7 @@ func (c *ClusterUpConfig) determineServerIP(out io.Writer) (string, []string, er
 
 // updateNoProxy will add some default values to the NO_PROXY setting if they are not present
 func (c *ClusterUpConfig) updateNoProxy() {
-	values := []string{"127.0.0.1", c.ServerIP, "localhost", openshift.ServiceCatalogServiceIP, registry.RegistryServiceClusterIP}
+	values := []string{"127.0.0.1", c.ServerIP, "localhost", service_catalog.ServiceCatalogServiceIP, registry.RegistryServiceClusterIP}
```

Comment:
wish we can pass this somehow to the components so they can append here...

Reply:
(but that will be tricky as appending to this list during runtime will mean you have to restart the container to pickup the new environment variable values)

Reply:
> wish we can pass this somehow to the components so they can append here...

We should remove it and see what happens.

Reply:
@deads2k people with weird proxies happen :-)

Reply:
> @deads2k people with weird proxies happen :-)

But the pattern of "expose all the services you'll ever have via no-proxy" doesn't scale